python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,876 @@
+      "text/plain": [
+       "<Figure size 720x936 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def groupBy():\n",
+    "    filter='landuse=forest or natural=wood'\n",
+    "    resU = users(\"/count/groupBy/boundary\"\n",
+    "         ,filter=filter          \n",
+    "         ,bpolys=regions\n",
+    "         ,time=t_mon)\n",
+    "    display(resU.url)\n",
+    "    green_u = resU.json()\n",
+    "    #display(green_u)\n",
+    "    df = pd.DataFrame()\n",
+    "    display(df)\n",
+    "    for groupBy in green_u['groupByResult']:\n",
+    "        series = pd.Series({row['fromTimestamp']:row['value'] for row in groupBy['result']})\n",
+    "        df[groupBy['groupByObject']] = series\n",
+    "    df.set_index(pd.to_datetime(df.index),inplace=True)\n",
+    "    df.to_pickle(\"./df_users.pkl\")\n",
+    "    df.plot(kind='line', subplots=True, sharex=False, sharey=True, figsize=(10,13), fontsize=9)\n",
+    "groupBy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "revolutionary-sigma",
+   "metadata": {},
+   "source": [
+    "#### _In the graphic above you can see the user numbers of the natural=wood/landuse=forest tag over the years_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "confidential-purchase",
+   "metadata": {},
+   "source": [
```

Comment:
I'd say here you can reduce the content a bit. Try to focus on the important parts that you want to hightlight. If there is nothing special about the graph of one city, then just mention it with one short sentence.